main.js

```diff
@@ -768,6 +773,7 @@ async function createWindow() {
                         infoPlayerProvider.getPlayerInfo().likeStatus !==
                         'DISLIKE'
                     ) {
+                        1257
```

Comment:
What's this `1257` for? 🤣

Reply:
I do not remember adding that actually, uhhh
it was probably me trying to type something in another window but forgot vscode was still in focus, and didn't realize i put that there

Reply:
Just fixed it

Reply:
Cool! It happens sometime for me as well. Lol